filtered_registries/simple_standard_scenarios/configs/projects/dsgrid_conus_2022/1.3.0/project.toml

```diff
@@ -2,7 +2,6 @@ project_id = "dsgrid_conus_2022"
 name = "dsgrid CONUS 2022"
 description = "Dataset created for the FY21 dsgrid Load Profile Tool for Grid Modeling project"
 status = "Complete"
-dimension_associations = [ "data_source__metric.csv", "data_source__sector.csv", "data_source__subsector.csv",]
```

Comment:
This file is normally auto-generated. I removed this line so that the file would load and we can test the query code (this is used for queries and not registration/dataset validation). The normal order is to merge the dsgrid code, update the SS repository, and then regenerate this filtered registry.